functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,167 @@
+# apply-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Apply setter values on resource fields. Setters serve as parameters for template-free
+setting of field values.
+
+<!--mdtogo-->
+
+Setters are a safer alternative to other substitution techniques which do not have the context
+of the structured data. Setters may be invoked to modify the package resources
+using `apply-setters` function to set values.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Package publishers declare setters in the package, consumers can set their values
+either declaratively or imperatively.
+
+Setter names can be discovered in the pipeline section of Kptfile, and the values
+can be declared next to setter names.
+
+```yaml
+apiVersion: v1alpha2
+kind: Kptfile
+metadata:
+  name: my-pkg
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fns/apply-setters:unstable
```

Comment:
You're using `unstable` images throughout the usage doc. Related to whether we want to show kpt CLI usage here.

Reply:
What should I use instead ? Just a place holder ? `gcr.io/kpt-fns/apply-setters:VERSION` ?

Reply:
The usage is tied to a particular version, so treat it like how you treat the source code. The question is what's the process for changing this version as the function is versioned?

@mengqiy This is part of the larger versioning story which also include example docs.

There's another discussion (I made a separate comment but GH comment UI sucks): Dependency on kpt CLI in such usage docs. We need to have a clear guideline on that, and include it in the usage template. Do we want each function to talk about imperative versus declarative workflow? We might want to meet up to finalize these decisions.

Reply:
Phani can you add TODOs with the tracking issue. We can't have `unstable` shown to users.